packages/mobile/src/store/store.ts

```diff
@@ -57,29 +58,35 @@ export type AppState = CommonState & {
   walletConnect: WalletConnectState
   shareToStoryProgress: ShareToStoryProgressState
 }
-
 const commonStoreReducers = commonReducers()
 
-const createRootReducer = () =>
-  combineReducers({
-    ...commonStoreReducers,
-    // These also belong in common store reducers but are here until we move them to the @audius/common package:
-    backend,
-    confirmer,
-    signOn: signOnReducer as SignOnPageReducer,
-    searchBar,
+const rootReducer = combineReducers({
+  ...commonStoreReducers,
+  // These also belong in common store reducers but are here until we move them to the @audius/common package:
+  backend,
+  confirmer,
+  signOn: signOnReducer as SignOnPageReducer,
+  searchBar,
+
+  drawers,
+  downloads,
+  keyboard,
+  oauth,
+  offlineDownloads,
+  remoteConfig,
+  search,
+  mobileUi,
+  walletConnect,
+  shareToStoryProgress
+})
 
-    drawers,
-    downloads,
-    keyboard,
-    oauth,
-    offlineDownloads,
-    remoteConfig,
-    search,
-    mobileUi,
-    walletConnect,
-    shareToStoryProgress
-  })
+// const persistConfig = {
```

Comment:
Did you leave this here intentionally?

Reply:
no i didnt! i thought initially you had to persist the root all the way down, but looks like can just add it anywhere, so clutch